src/components/services-view/services-view.html

```diff
@@ -44,5 +45,16 @@
         </div>
       </div>
     </div>
+
+    <div ng-if="$ctrl.currentFilter === 'other'"
+         class="sub-category-row other"
+         pf-card-view config="$ctrl.cardViewConfig" items="$ctrl.filteredItems">
+      <div class="card-container">
+        <div ng-if="!item.imageUrl" class="card-icon {{item.iconClass}}"></div>
+        <div ng-if="item.imageUrl" class="card-img"><img src="{{item.imageUrl}}"></div>
+        <div class="card-name">{{item.name}}</div>
+      </div>
+    </div>
+
   </div>
```

Comment:
I guess I'm not clear why it's different at all. Shouldn't it just be another group and the items included in "All"?